recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -1,13 +1,13 @@
 from conan import ConanFile
-from conan.tools import files
 from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, save, copy, apply_conandata_patches
```

Comment:
```suggestion
from conan.tools.files import get, save, copy, apply_conandata_patches, export_conandata_patches
```
@talyz Now 1.52.0 is running on CCI, you can use export_conandata_patches 😂